CHANGELOG.md

```diff
@@ -12,6 +12,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - added new method `Grid::delete_orders` and the corresponding switch
   `--delete-orders` in the subcommand `write` of the CLI
 
+### Changed
+
+- Python API: dropped top-level Python interface layer
+- Python API: renamed `lumi` to `channel` in PyO3 Python interface. This
+  concerns 1) the argument names of `convolute_with_one` and similar functions;
+  2) the module `pineappl.lumi` was moved to `pineappl.channel`; 3) the class
```

Comment:
```suggestion
  2) the module `pineappl.lumi` was moved to `pineappl.boc`; 3) the class
```
should we add more details to the changelog?

Reply:
Yes, that one should definitely be fixed. Maybe @cschwan wants to add more information there?

Reply:
Good catch!